miner/worker.go

```diff
@@ -95,10 +98,11 @@ type worker struct {
 	pendingLogsFeed event.Feed
 
 	// Subscriptions
-	mux      *event.TypeMux // TODO replace
-	mu       sync.RWMutex   // The lock used to protect the coinbase and extra fields
-	coinbase common.Address
-	clock    *mockable.Clock // Allows us mock the clock for testing
+	mux        *event.TypeMux // TODO replace
+	mu         sync.RWMutex   // The lock used to protect the coinbase and extra fields
+	coinbase   common.Address
+	clock      *mockable.Clock // Allows us mock the clock for testing
+	beaconRoot *common.Hash    // XXX: this is not set anywhere
```

Comment:
I think we should support setting beaconRoot somehow during block building, but not sure what we would put in it.